client/my-sites/site-settings/controller.js

```diff
@@ -64,8 +64,7 @@ module.exports = {
 	siteSettings( context ) {
 		let analyticsPageTitle = 'Site Settings';
 		const basePath = route.sectionify( context.path );
-		const fiveMinutes = 5 * 60 * 1000;
-		let site = sites.getSelectedSite();
+		const site = getSelectedSite( context.store.getState() );
```

Comment:
WIth a clean redux state this will break the `site.*` calls below. We should either guard against these calls if `site` is `null` (by specifically prepending a `site` check), or use selectors for all of those calls.

![](https://cldup.com/TBGPhzbBdT.png)